src/PointwiseFunctions/Hydro/EquationsOfState/EquationOfState.hpp

```diff
@@ -311,12 +311,18 @@ class EquationOfState<IsRelativistic, 1> : public PUP::able {
   /// The lower bound of the specific internal energy that is valid for this EOS
   /// at the given rest mass density \f$\rho\f$
   virtual double specific_internal_energy_lower_bound(
-      double rest_mass_density) const = 0;
+      double rest_mass_density) const {
```

Comment:
I think this might break @jyoo1042 's simulations with a 1d EOS. I believe for primitive recovery we need the thermal part effectively unrestricted and trying to restrict it caused the simulations to fail. Isn't what you would like for the 2d EOS to have a density-dependent bound, not the 1d EOS? For a 1d EOS the thermal part doesn't mattr